src/main/scala/fly/play/aws/AwsCredentials.scala

```diff
@@ -18,12 +18,14 @@ object AwsCredentials extends ((String, String, Option[String]) => AwsCredential
   def apply(accessKeyId: String, secretKey: String, token: Option[String] = None): AwsCredentials =
     SimpleAwsCredentials(accessKeyId, secretKey, token)
 
-  implicit def fromConfiguration(implicit app: Application): AwsCredentials = {
+  implicit def fromConfiguration(implicit app: Application): AwsCredentials = fromConfiguration("aws")
+
+  implicit def fromConfiguration(prefix: String = "aws")(implicit app: Application): AwsCredentials = {
```

Comment:
I think you're right about it not needing to be an implicit and the default isn't necessary as well. I'll update the code.